File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //Prompt user for height
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8); //height must be between 1 and 8 inclusive

    //Print image
    for (int i = 1; i <= height; i++)
    {
        for (int j = 0; j < height - i; j++) //print height-i blanks
        {
            printf(" ");
        }
        for (int k = 0; k < i; k++) //print number of #
        {
            printf("#");
        }
        printf("\n"); //new line
    }
}

File: sentimental-mario-less/mario.py
from cs50 import get_int

while True:  # prompt user for height
    num = get_int("Height: ")
    if num > 0 and num < 9:
        break

for i in range(0, num, 1):  # double 'for' loop to print height
    for j in range(0, num, 1):
        if i + j < num - 1:
            print(" ", end="")
        else:
            print("#", end="")
    print()
